sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/DicomIO.java

```diff
@@ -37,9 +36,27 @@
 /**
  * The DicomIO connectors allows Beam pipelines to make calls to the Dicom API of the Google Cloud
  * Healthcare API (https://cloud.google.com/healthcare/docs/how-tos#dicom-guide).
+ *
+ * <h3>Reading Study-Level Metadata</h3>
+ *
+ * The study-level metadata for a dicom instance can be read with {@link ReadDicomStudyMetadata}.
+ * Retrieve the metadata of a dicom instance given its store path as a string. This will return a
+ * {@link ReadDicomStudyMetadata.Result}. You can fetch the successful calls using
+ * getReadResponse(), and any failed reads using getFailedReads().
+ *
+ * <h3>Example</h3>
+ *
+ * {@code Pipeline p = ... String webPath = ... DicomIO.ReadDicomStudyMetadata.Result
+ * readMetadataResult = p .apply(Create.of(webPath)) PCollection<String> goodRead =
+ * readMetadataResult.getReadResponse() PCollection<String> failRead =
+ * readMetadataResult.getFailedReads() }
  */
 public class DicomIO {
 
+  public static ReadDicomStudyMetadata readDicomStudyMetadata() {
```

Comment:
ok one more question - does it make sense to just call it `readStudyMetadata` since we're already in DicomIO? Up to you.

Reply:
Makes sense to me